source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -48,7 +48,7 @@ public class ForestGameArea extends GameArea {
   private static final GridPoint2 KANGAROO_BOSS_SPAWN = new GridPoint2(25, 10);
   private static final float WALL_WIDTH = 0.1f;
   private final TerrainFactory terrainFactory;
-  private final List<Entity> enemies;
+  public static List<Entity> enemies = null;
```

Comment:
I wouldn't set it to null if you're going to set it to a new ArrayList in the initialiser anyways